### PR TITLE
Update references to AdoptOpenJDK to Adoptium

### LINKED ISSUE #2558: Update references to AdoptOpenJDK to Adoptium
There are some references to Github organizations and repos that need updating in this repository.  Now that all test materials are moved to Adoptium, we should search for and replace the references to any that are no longer valid (even though Github forwarding will mostly handle it, we should update nonetheless).  Org names and repo names need to be searched for and updated.

Test repos moved/renamed:
adoptopenjdk/openjdk-tests -> adoptium/aqa-tests
adoptopenjdk/openjdk-systemtest -> adoptium/aqa-systemtest
adoptopenjdk/openjdk-test-tools -> adoptium/aqa-test-tools
adoptopenjdk/STF -> adoptium/STF
adoptopenjdk/TKG -> adoptium/TKG
adoptopenjdk/bumblebench -> adoptium/bumblebench
adoptopenjdk/run-aqa -> adoptium/run-aqa

Other potentially referenced repos moved/renamed:
adoptopenjdk/build-jdk -> adoptium/build-jdk

### PR DESCRIPTION
**Pull request by 2001asjad**
Update references to AdoptOpenJDK to Adoptium wherever they were necessary. 

The changes were made to the following:-
Test repos moved/renamed:
adoptopenjdk/openjdk-tests -> adoptium/aqa-tests
adoptopenjdk/openjdk-systemtest -> adoptium/aqa-systemtest
adoptopenjdk/openjdk-test-tools -> adoptium/aqa-test-tools
adoptopenjdk/STF -> adoptium/STF
adoptopenjdk/TKG -> adoptium/TKG
adoptopenjdk/bumblebench -> adoptium/bumblebench
adoptopenjdk/run-aqa -> adoptium/run-aqa

Other potentially referenced repos moved/renamed:
adoptopenjdk/build-jdk -> adoptium/build-jdk

Fixes #2558

### CODE REVIEW

**Comment by 2001asjad** [comment]
@smlambert @llxia Can you please take a look at this PR.
Looking forward to your analysis.

Thank You.

File: buildenv/jenkins/getDependency [hunk 1]
```diff
@@ -21,8 +21,8 @@ def testBuild() {
 				sh 'mkdir ${WORKSPACE}/j2sdk-image'
 				sh 'tar -xzf OpenJDK8U-jdk_x64_linux_hotspot*.gz -C ${WORKSPACE}/j2sdk-image --strip-components 1'
 				sh '${WORKSPACE}/j2sdk-image/jre/bin/java -version'
-				sh 'git clone https://github.com/AdoptOpenJDK/openjdk-systemtest'
-				sh 'git clone https://github.com/AdoptOpenJDK/stf'
+				sh 'git clone https://github.com/adoptium/aqa-systemtest'
+				sh 'git clone https://github.com/adoptium/STF'
 				sh 'ant -f ./openjdk-systemtest/openjdk.build/build.xml -Djava.home=${WORKSPACE}/j2sdk-image/jre -Dprereqs_root=${WORKSPACE}/systemtest_prereqs configure'
```
Comment (llxia) [inline buildenv/jenkins/getDependency:26]: `openjdk-systemtest` needs to be updated to `aqa-systemtest`. The same for line 27

File: buildenv/jenkins/openjdk_tests [hunk 2]
```diff
@@ -292,23 +292,28 @@ if (JDK_VERSIONS.size() > 1 || JDK_IMPLS.size() > 1 || PLATFORMS.size() >1 || PL
...
+                    }
+                    jenkinsfile = load "${WORKSPACE}/openjdk-tests/buildenv/jenkins/JenkinsfileBase"
+                    if (LABEL.contains('ci.agent.dynamic') && CLOUD_PROVIDER == 'azure') {
+                        //Set dockerimage for azure agent. Fyre has stencil to setup the right environment
+                        docker.image('adoptopenjdk/centos6_build_image').pull()
+                        docker.image('adoptopenjdk/centos6_build_image').inside {
+                            jenkinsfile.testBuild()
+                        }
+                    } else {
+
```
Comment (llxia) [inline buildenv/jenkins/openjdk_tests:316]: This change is not related to the git reference update. Did you change the indentation?

**Review by sophia-guo — APPROVED** [review]
LGTM

**Review by llxia — CHANGES_REQUESTED** [review]

File: buildenv/jenkins/openjdk_tests (hunk 2 above)
Comment (2001asjad) [reply buildenv/jenkins/openjdk_tests:316]: No @llxia I didn't change the indentation. 
Any suggestions?

Comment (2001asjad) [reply buildenv/jenkins/openjdk_tests:316]: This might have occurred while I was trying to resolve the conflicts. Shall I make it the same (to original)?

File: buildenv/jenkins/JenkinsfileBase [hunk 3]
```diff
@@ -430,7 +430,7 @@ def setup() {
 			GET_SH_CMD = "./openjdk-tests/get.sh -s `pwd` -t `pwd`/openjdk-tests -p $PLATFORM -r ${SDK_RESOURCE} ${JDK_VERSION_OPTION} ${JDK_IMPL_OPTION} ${CUSTOMIZED_SDK_URL_OPTION} ${CUSTOMIZED_SDK_SOURCE_URL_OPTION} ${CLONE_OPENJ9_OPTION} ${OPENJ9_REPO_OPTION} ${OPENJ9_BRANCH_OPTION} ${OPENJ9_SHA_OPTION} ${TKG_REPO_OPTION} ${TKG_BRANCH_OPTION} ${TKG_SHA_OPTION} ${VENDOR_TEST_REPOS} ${VENDOR_TEST_BRANCHES} ${VENDOR_TEST_DIRS} ${VENDOR_TEST_SHAS} ${TEST_IMAGES_REQUIRED} ${DEBUG_IMAGES_REQUIRED}"
```
Comment (llxia) [inline buildenv/jenkins/JenkinsfileBase:430]: `openjdk-tests` -> `aqa-tests`

File: buildenv/jenkins/openjdk_tests [hunk 4]
```diff
@@ -291,7 +291,7 @@ if (JDK_VERSIONS.size() > 1 || JDK_IMPLS.size() > 1 || PLATFORMS.size() >1 || PL
                     sh "git clone -b ${SCM_GIT_BRANCH} ${SCM_GIT_REPO_VAL} openjdk-tests"
                 } else {
                     def gitConfig = scm.getUserRemoteConfigs().get(0)
-
+                    
                     // Adopt windows machines require env here https://github.com/AdoptOpenJDK/openjdk-tests/issues/1803
```
Comment (llxia) [inline buildenv/jenkins/openjdk_tests:295]: Please updated line 295

File: buildenv/jenkins/openjdk_tests [hunk 5]
```diff
@@ -291,7 +291,7 @@ if (JDK_VERSIONS.size() > 1 || JDK_IMPLS.size() > 1 || PLATFORMS.size() >1 || PL
                     sh "git clone -b ${SCM_GIT_BRANCH} ${SCM_GIT_REPO_VAL} openjdk-tests"
```
Comment (llxia) [inline buildenv/jenkins/openjdk_tests:291]: I think we need to update openjdk-tests

File: buildenv/jenkins/JenkinsfileBase [hunk 6]
```diff
@@ -427,10 +427,10 @@ def setup() {
...
 			TEST_IMAGES_REQUIRED = (params.TEST_IMAGES_REQUIRED == false) ? "--test_images_required false" : ""
 			DEBUG_IMAGES_REQUIRED = (params.DEBUG_IMAGES_REQUIRED == false) ? "--debug_images_required false" : ""
-			GET_SH_CMD = "./openjdk-tests/get.sh -s `pwd` -t `pwd`/openjdk-tests -p $PLATFORM -r ${SDK_RESOURCE} ${JDK_VERSION_OPTION} ${JDK_IMPL_OPTION} ${CUSTOMIZED_SDK_URL_OPTION} ${CUSTOMIZED_SDK_SOURCE_URL_OPTION} ${CLONE_OPENJ9_OPTION} ${OPENJ9_REPO_OPTION} ${OPENJ9_BRANCH_OPTION} ${OPENJ9_SHA_OPTION} ${TKG_REPO_OPTION} ${TKG_BRANCH_OPTION} ${TKG_SHA_OPTION} ${VENDOR_TEST_REPOS} ${VENDOR_TEST_BRANCHES} ${VENDOR_TEST_DIRS} ${VENDOR_TEST_SHAS} ${TEST_IMAGES_REQUIRED} ${DEBUG_IMAGES_REQUIRED}"
+			GET_SH_CMD = "./aqa-tests/get.sh -s `pwd` -t `pwd`/openjdk-tests -p $PLATFORM -r ${SDK_RESOURCE} ${JDK_VERSION_OPTION} ${JDK_IMPL_OPTION} ${CUSTOMIZED_SDK_URL_OPTION} ${CUSTOMIZED_SDK_SOURCE_URL_OPTION} ${CLONE_OPENJ9_OPTION} ${OPENJ9_REPO_OPTION} ${OPENJ9_BRANCH_OPTION} ${OPENJ9_SHA_OPTION} ${TKG_REPO_OPTION} ${TKG_BRANCH_OPTION} ${TKG_SHA_OPTION} ${VENDOR_TEST_REPOS} ${VENDOR_TEST_BRANCHES} ${VENDOR_TEST_DIRS} ${VENDOR_TEST_SHAS} ${TEST_IMAGES_REQUIRED} ${DEBUG_IMAGES_REQUIRED}"
```
Comment (llxia) [inline buildenv/jenkins/JenkinsfileBase:430]: openjdk-tests -> aqa-tests

**Comment by llxia** [comment]
Thanks @2001asjad 
The ECA check failed. Did you sign the Eclipse Contributor Agreement (ECA)?https://www.eclipse.org/legal/ECA.php

**Comment by llxia** [comment]
Do you have access to Grinder? If so, can you run a Grinder to test your PR?

**Comment by 2001asjad** [comment]
> Thanks @2001asjad
> The ECA check failed. Did you sign the Eclipse Contributor Agreement (ECA)?https://www.eclipse.org/legal/ECA.php

Yeah. I have signed in

**Comment by llxia** [comment]
In the future, please do not reformat the entire file. It is very hard to see what is actually changed.

**Comment by llxia** [comment]
Re https://github.com/adoptium/aqa-tests/pull/2607#issuecomment-853257468, please see the error https://accounts.eclipse.org/legal/eca/validation/69380

```
Asjad Khan (iitasjad2001@****.com) is not covered by the necessary legal agreements in order to proceed.
Asjad Khan (60435499+2001asjad@****.noreply) is not covered by the necessary legal agreements in order to proceed.
```

**Comment by 2001asjad** [comment]
> In the future, please do not reformat the entire file. It is very hard to see what is actually changed.

Sorry for that.. After the LGTM suggestion.. Things got more complicated.. I have signed in and filled the ECA form as well.. 
Let me take a look again and I'm gonna update you very soon
